docker/api/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM gorghoa/php7
```

Comment:
Once available, we could definitely use the api-platform base image. But for now, I think it’s not a blocker for this PR.